### DISCUSSION TITLE
Login session expired

### DISCUSSION THREAD

**Discussion by mizzunet** [dani-garcia/vaultwarden#1767] [Q&A]
Nevermind

**Comment by jjlin** [comment]
You shouldn't be restoring backups on startup. If you mount `/vw-data` on your Docker host as `/data` in the container, the data will still be there after the container restarts.

**Reply by mizzunet** [reply]
Yeah, you are right. Well, what about the issue?

**Reply by jjlin** [reply]
Maybe the client and server interact after your restore, so you end up restoring something not entirely up to date. Hard to say without being able to see exactly what's happening, but don't do weird stuff and you'll probably avoid having weird problems.

**Comment by BlackDex** [comment]
Probably the time and date are not correct.

**Reply by mizzunet** [reply]
Phone's? It's not. Pi's I just cross-checked and that also correct

**Comment by mizzunet** [comment]
Btw, I use Agro Tunnel for making it available online

**Comment by BlackDex** [comment]
Just as a note here (Because you probably have your answer already via the matrix chat).
It could also be that the RSA key's aren't restored, if they are re-created, then all the tokens are invalid.

**Reply by mizzunet** [reply]
I do backup rsa keys as well https://github.com/dani-garcia/vaultwarden/wiki/Backing-up-your-vault#the-rsa_key-files.

Btw sorry I lied about Pi earlier :P
